### PR TITLE
[OCPCLOUD-1854]: Update k8s to 1.26

### PR DESCRIPTION
**Pull request by odvarkadaniel**
To align with the 4.13 release, k8s dependencies need to be updated to 1.26

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -13,23 +13,23 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/hashicorp/golang-lru v0.5.1
 	github.com/mitchellh/mapstructure v1.4.2
-	github.com/onsi/ginkgo/v2 v2.7.0
-	github.com/onsi/gomega v1.24.1
+	github.com/onsi/ginkgo/v2 v2.8.1
+	github.com/onsi/gomega v1.26.0
 	github.com/openshift/api v0.0.0-20220921125526-1866ef90edbf
 	github.com/openshift/machine-api-operator v0.2.1-0.20220922080915-f7f326323f37
```
Comment (JoelSpeed) [inline go.mod:19]: Are these both up to date?

Comment (odvarkadaniel) [reply go.mod:19]: They are not. Do we want to update them? Ran into some issue while doing so.

Comment (JoelSpeed) [reply go.mod:19]: Yeah it would be good to update them if we can, what issues came up?

Comment (odvarkadaniel) [reply go.mod:19]: Looked into this and both of these should be up to date.

Comment (JoelSpeed) [reply go.mod:19]: Have you pushed? They both look like they were updated in september?

Comment (odvarkadaniel) [reply go.mod]: Updated.

**Pull request merged**